src/collection-preferences/content-display/content-display-option.scss

```diff
@@ -21,60 +21,54 @@ $border-radius: awsui.$border-radius-item;
   }
 }
 
-.sortable-item-toggle {
+.content-display-option-toggle {
```

Comment:
Harmonized internal names with testing utils API: "sortable item" is now "content display option".